python/cudnn/sdpa/fwd/kernels/prefill_d512_f16_sm100.py

```
@@ -1874,7 +1882,18 @@ def _tma_swz(byte_w: int):
 
 
 @lru_cache(maxsize=None)
-def compile(b: int = 1, qh: int = 1, kh: int = 1, sq: int = 256, skv: int = 128, d_qk: int = CFG.TILE_K, d_v: int = CFG.TILE_O) -> Callable:
+def compile(
```

Ruff (0.16.1)

[error] Variable `compile` is shadowing a Python builtin

(A001)